tools/wpt/browser.py

```diff
@@ -1230,19 +1230,12 @@ def find_binary(self, venv_path=None, channel=None):
             return path
 
         if uname[0] == "Linux":
-            name = "google-chrome"
-            if channel == "stable":
-                name += "-stable"
-            elif channel == "beta":
-                name += "-beta"
-            elif channel == "dev":
-                name += "-unstable"
-            # No Canary on Linux.
-            return which(name)
+            return which("chrome")
```

Comment:
Does this also work with other channels?

Reply:
Yes - all versions of the binary are named "chrome" using Chrome for Testing on Linux.